### DISCUSSION TITLE
An option to hide the clock emoji in menubar

### DISCUSSION THREAD

**Discussion by ShunKaiser** [DynamicPlayerSector/timethings#12] [Ideas]
Hi @DynamicPlayerSector 

I enjoy utilizing your plugin, particularly its unique feature of automatically tracking editing time. However, I've encountered a minor issue with the noticeable clock emoji in the menu bar. Since I typically dim the menu bar and focus on word counts and backlinks, the colored emoji stands out. It would be great if, in the next update, you could include an option to hide it. The "duration key name" is enough for me to see the information. This adjustment would make the plugin perfect for my needs.

Thank you!

**Comment by plasmabit** [comment]
If you want to hide the whole clock, there is already an option for that: "Status Bar" -> "Enable status bar clock".

**Comment by ShunKaiser** [comment]
> If you want to hide the whole clock, there is already an option for that: "Status Bar" -> "Enable status bar clock".

Oh I apologize that it wasn't clear enough. I mean the hourglass emoji ⌛️. Is there a way to hide it?

**Comment by plasmabit** [comment]
Added a new feature to hide emojis in 1.2.6

https://github.com/DynamicPlayerSector/timethings/releases/tag/1.2.6
